test/cli_create_log_options_test.go

```diff
@@ -78,7 +78,7 @@ func (suite *PouchCreateLogOptionsSuite) TestOK(c *check.C) {
 			driver:   "json-file",
 			expected: nil,
 		}, {
-			cname:   "TestCreateLogOptions_jsonfile_tag=1",
+			cname:   "TestCreateLogOptions_jsonfile_tag_1",
```

Comment:
why change that?

Reply:
seems it verify condition that logOpts: []string{"tag=1"}?

Reply:
Creating a container name with an equal sign is illegal.